tests/uc_oss/test_uc_oss_integration.py

```diff
@@ -0,0 +1,152 @@
+import os
+import subprocess
+import sys
+
+import pandas as pd
+import pytest
+from sklearn import datasets
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.model_selection import train_test_split
+
+import mlflow
+
+from tests.helper_functions import get_safe_port
+from tests.tracking.integration_test_utils import _await_server_up_or_die
+
+pytestmark = pytest.mark.skipif(
+    "UC_OSS_INTEGRATION" not in os.environ,
+    reason="This test is only valid w/in the github workflow integration job",
+)
+
+
+@pytest.fixture(scope="module")
+def setup_servers():
+    port = get_safe_port()
+    with (
+        subprocess.Popen(
+            ["bin/start-uc-server"],
+            cwd="unitycatalog",
+        ) as uc_proc,
+        subprocess.Popen(
+            [sys.executable, "-m", "mlflow", "server", "--port", str(port)]
+        ) as mlflow_proc,
+    ):
+        try:
+            _await_server_up_or_die(int(port))
+            _await_server_up_or_die(8080)
+
+            mlflow_tracking_url = f"http://127.0.0.1:{port}"
+            uc_oss_url = "uc:http://127.0.0.1:8080"
+
+            mlflow.set_tracking_uri(mlflow_tracking_url)
+            mlflow.set_registry_uri(uc_oss_url)
+
+            yield mlflow_tracking_url
+        finally:
+            mlflow_proc.terminate()
+            uc_proc.terminate()
+
+
+def test_integration(setup_servers):
+    catalog = "unity"
+    schema = "default"
+    registered_model_name = "iris"
+    model_name = f"{catalog}.{schema}.{registered_model_name}"
+    mlflow.set_experiment("iris-uc-oss")
+    client = mlflow.MlflowClient()
+    with pytest.raises(Exception, match="NOT_FOUND"):
+        client.get_registered_model(model_name)
+
+    X, y = datasets.load_iris(return_X_y=True, as_frame=True)
+    X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.2, random_state=42)
+
+    def build_model():
+        with mlflow.start_run():
+            # Train a sklearn model on the iris dataset
+            clf = RandomForestClassifier(max_depth=7)
```

Comment:
```suggestion
    with mlflow.start_run():
        # Train a sklearn model on the iris dataset
        clf = RandomForestClassifier(max_depth=7)
```

let's run this directly